### PR TITLE
Make it less opinionated

### LINKED ISSUE #3: Stop depending on `express-statsd`

### PR DESCRIPTION
**Pull request by jfresco**
- Remove `express-statsd` dependency
- Make behavior less opinionated, more configurable
- Expose statsd client
- Update readme

It's a breaking change.
Closes #3 #4 #1

### CODE REVIEW

File: dist/helpers.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+'use strict';
```
Comment (gschammah) [inline dist/helpers.js]: es necesario commitear esto?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 *.log
 node_modules
+dist
```
Comment (gschammah) [inline .gitignore]: detalle: faltaria el new line

File: .npmignore [hunk 3]
```diff
@@ -0,0 +1,10 @@
+*.swp
+*~
+*.iml
+.*.haste_cache.*
+.DS_Store
+.idea
```
Comment (gschammah) [inline .npmignore]: solo incluiria archivos comiteados. muchos de ellos no lo estan

File: src/helpers.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  }
+}
+
+/**
+ * Gets the time in ms since the request started
+ *
+ * @return {Number} Time in ms since the request started
+ */
+
+export const getRequestDuration = startTime => new Date().getTime() - startTime
```
Comment (gschammah) [inline src/helpers.js:31]: no deberiamos usar https://nodejs.org/docs/v0.8.0/api/all.html#all_process_hrtime?

Comment (jfresco) [reply src/helpers.js:31]: `process#hrtime` gives you the diff in nanoseconds. We need the duration in ms, so we have to convert it.
That substraction gives you the diff in ms. I don't know if it is less precise, but for sure we don't have the (possible?) error in the conversion.

**Comment by gschammah** [comment]
👍

**Pull request merged**